src/platform/plugins/shared/unified_doc_viewer/tsconfig.json

```diff
@@ -1,9 +1,14 @@
 {
   "extends": "../../../../../tsconfig.base.json",
   "compilerOptions": {
-    "outDir": "target/types",
+    "outDir": "target/types"
   },
-  "include": [ "../../../../../typings/**/*", "common/**/*", "public/**/*", "server/**/*"],
+  "include": [
+    "common/**/*",
+    "public/**/*",
+    "server/**/*",
+    "typings/**/*",
```

Comment:
```suggestion
```
Nit: we should be able to drop this now that the `typings` folder has been moved to `public`.